client/my-sites/themes/current-theme/index.jsx

```diff
@@ -46,11 +46,19 @@ class CurrentTheme extends Component {
 			currentTheme && ! ( option.hideForTheme && option.hideForTheme( currentTheme, siteId ) )
 		);
 
+		const showScreenshot = currentTheme && currentTheme.screenshot;
+		// Show nothing for themes with no screenshot
+		const screenshotPlaceholder = ! currentTheme;
```

Comment:
> maybe tie it to `showScreenshot` like
> `screenshotPlaceholder = ! showScreenshot`

Some (custom) themes have no screenshot, so the above line would leave a flashing placeholder. I'll improve the comment to try and make that clearer.